Cargo.toml

```diff
@@ -83,7 +83,7 @@ debug = true
 debug = true
 
 [patch.crates-io]
-halo2_gadgets = { git = "https://github.com/zcash/halo2.git", rev = "30f92f3f4b785ea2a32392bf65c1b08f0411567c" }
-halo2_proofs = { git = "https://github.com/zcash/halo2.git", rev = "30f92f3f4b785ea2a32392bf65c1b08f0411567c" }
+halo2_gadgets = { git = "https://github.com/zcash/halo2.git", rev = "4b802a7d07b4eefeca137e4f0939ce8e13e0b954" }
+halo2_proofs = { git = "https://github.com/zcash/halo2.git", rev = "4b802a7d07b4eefeca137e4f0939ce8e13e0b954" }
```

Comment:
Checked that these match the upstream PR.

Reply:
Actually we'll want to change these to the post-merge commit hashes.

Reply:
Done.